### PR TITLE
Allow message.fatal to be null

### PR DESCRIPTION
**Pull request by brendo**
Based off #17, this allows a user to create a message that has the indeterminate fatal value of `null`.

### CODE REVIEW

**Comment by wooorm** [comment]
How about a new `.info` method, like `.fail`, that sets `fatal: null`?

Reasoning: so this doesn’t need to be a major version, and so that we don’t have four (quite a lot) parameters in `.message`?

**Comment by brendo** [comment]
I like it, wasn't fond of the method signature either. Will amend.

**Comment by brendo** [comment]
Back with you @wooorm 👍

File: index.js [hunk 1]
```diff
@@ -153,8 +154,8 @@ function toString(encoding) {
 }
 
 /* Create a message with `reason` at `position`.
- * When an error is passed in as `reason`, copies the
- * stack.  This does not add a message to `messages`. */
+ * When an error is passed in as `reason`, copies the stack.
+ * Errors are not added as a message to `messages`. */
```
Comment (wooorm) [inline index.js]: I guess this line (`Errors are not added as a message to `messages`.`) no longer applies? Could you remove it?

File: index.js [hunk 2]
```diff
@@ -212,6 +213,16 @@ function fail() {
...
+/* Info. Creates a vmessage, associates it with the file,
+ * and marks the fatality as null. */
+function info() {
+  var message = this.message.apply(this, arguments);
+
+  message.fatal = null;
+
+  return message;
+}
+
```
Comment (wooorm) [inline index.js]: ✅

File: test.js [hunk 3]
```diff
@@ -502,6 +502,32 @@ test('vfile([options])', function (t) {
...
+    st.equal(message.line, 1);
+    st.equal(message.column, 3);
+    st.deepEqual(message.location, {
+      start: {line: 1, column: 3},
+      end: {line: null, column: null}
+    });
+
+    st.end();
+  });
+
```
Comment (wooorm) [inline test.js]: ✅

**Comment by wooorm** [comment]
Looks good, could you remove that one referenced line, and include docs in the readme? It’s all ✅ if that’s done!

**Comment by brendo** [comment]
Done @wooorm, sorry I should've done that to begin with!

**Comment by wooorm** [comment]
Thanks so much Brendan for working on this! 👏

**Comment by brendo** [comment]
Awesome, thank you!

**Pull request closed without merging**